docs/architecture/SDR_SERVICE_ARCHITECTURE.md

```
@@ -136,10 +136,10 @@ Format: JSON with zlib+base64 encoded samples
 }
 ```
 
-### Spectrum Data (Waterfall)
+### Spectrum Data (Waterfall / Spectrum Scope)
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)